spec/adhearsion/configuration_spec.rb

```diff
@@ -98,16 +98,16 @@
       end
 
       before do
-        subject.production do |env|
+        subject.env(:production) do |env|
```

Comment:
Do we really need the env prefix? Can't we just evaluate the block only when the environment matches?